deploy/crds/app.kiegroup.org_kogitoapps_crd.yaml

```diff
@@ -206,49 +369,25 @@ spec:
               description: 'Number of replicas that the service will have deployed
                 in the cluster Default value: 1'
               format: int32
-              maximum: 100
               minimum: 0
               type: integer
             resources:
-              description: 'The resources for the deployed pods, like memory and cpu
-                Default value: nil'
+              description: Defined Resources for the Jobs Service
               properties:
                 limits:
-                  items:
-                    description: ResourceMap Data to define a list of possible Resources
-                    properties:
-                      resource:
-                        description: Resource type like CPU and memory
-                        enum:
-                        - cpu
-                        - memory
-                        type: string
-                      value:
-                        description: Value of this resource in Kubernetes format
-                        type: string
-                    required:
-                    - resource
-                    - value
-                    type: object
-                  type: array
+                  additionalProperties:
+                    type: string
+                  description: 'Limits describes the maximum amount of compute resources
+                    allowed. More info: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/'
+                  type: object
```

Comment:
The requests and limits structure in CRD is generated, right?
Because from this description it is hard to guess the correct CR syntax, which seems to be for example
```
...
    resources:
      requests:
        memory: "4Gi"
      limits:
        memory: "4Gi"
...
```

Reply:
It's auto generated based on the kubernetes Resource, we can't modify this description.